src/test/java/org/kairosdb/core/health/DatastoreQueryHealthCheckTest.java

```diff
@@ -26,15 +26,14 @@
     public void setup() throws DatastoreException {
```

Comment:
Can you add test(s) where
* Tags is nonempty and valid
* Tags is nonempty and invalid